data/transition-sites/beis_energyadvice.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: beis_energyadvice
+whitehall_slug: department-for-business-energy-and-industrial-strategy
+homepage: https://www.gov.uk/government/organisations/department-for-business-energy-and-industrial-strategy
+tna_timestamp: 20220610155057
+host: www.simpleenergyadvice.org.uk
+global: =301 https://www.gov.uk/improve-energy-efficiency
```

Comment:
I think this makes it redirect for all paths on the site, so we probably want to remove this line as they want mappings